src/lib/dom-api-shadow.html

```diff
@@ -58,6 +58,15 @@
 
   Object.defineProperties(DomApi.prototype, {
 
+    activeElement: {
+      get: function() {
+        var node = Polymer.DomApi.wrap(this.node);
+        var activeElement = node.activeElement;
+        return node.contains(activeElement) ? activeElement : null;
```

Comment:
Let's add a comment here that explains the `contains` check. I believe this is so that we ignore distributed nodes. We should also link to the spec bug where the native behavior is changing.